virtcontainers/clh.go

```diff
@@ -46,6 +46,8 @@ const (
 
 const (
 	clhTimeout            = 10
+	clhApiTimeout         = 1
```

Comment:
I feel that 3  or 4 would give a  more flexibility in general, and still we prevent any hang.

Reply:
Sure. I trust your assessment. Please feel free to adjust it in your code base.